data_structures/linked_list/QueueLinkedList.c

```diff
@@ -0,0 +1,133 @@
+#include<stdio.h>
```

Comment:
My humble request is that you please compile the code that you write before pushing it to this PR. I am finding compile-time errors/warnings which you can fix beforehand.